src/stories/components/svg/IconOpenModal.tsx

```diff
@@ -5,12 +5,14 @@ interface Props {
   width?: number;
   height?: number;
   onClick?: () => void;
+  className?: string;
 }
 
-const IconOpenModal: React.FC<Props> = ({ height = 17, width = 16, onClick, ...props }) => {
+const IconOpenModal: React.FC<Props> = ({ height = 17, width = 16, onClick, className, ...props }) => {
```

Comment:
we should stop using these icons and export the SVG to the public icon, add it to the iconography in the storybook and use it here

Reply:
In this case this a work in progress, this will be done when the items relate with the icon solved